omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -73,6 +73,7 @@ def run!
     begin
       PostgresqlPreflightValidator.new(node).run!
       SolrPreflightValidator.new(node).run!
+      BookshelfPreflightValidator.new(node).run!
```

Comment:
I definitely love that we're continuing to add preflight checks for things that we know are either unsupported or just won't work. Nice job. :100: